src/README.md

```diff
@@ -32,9 +32,16 @@ var myApp = new springroll.Application({
     buttonSize: false, // whether or not the game has adjustable button sizes
     removableLayers: false, // whether or not the game supports the removal of distracting game layers
     hudPosition: false, // whether or not the game supports multiple HUD positions
-    difficulty: false, // whether or not the game supports an adjustable difficulty,
-    keyBinding: true, //whether or not the game supports re-mappable key bindings.
-    colorVision: true, //whether or not the game supports alternate color schemes for color blind users
+    hitAreaScale: true, // whether or not the game supports adjustable hit areas
```

Comment:
I think we should also add a section to the README to provide some examples of when these mechanics should be configurable. Some thing ala:

# Hit Area Scale
Set this feature to true if your game supports scalable hit boxes for items in your game.

☝️ Something like that.

Reply:
I've added documentation for these difficulty features to the read me